### DISCUSSION TITLE
add more badges?

### DISCUSSION THREAD

**Discussion by Atoxiv** [SevenTV/chatterino7#266] [Ideas]
Maybe add BTTV Badges and FFZ:AP Badges to chatterino aswell? :3

![image](https://github.com/SevenTV/chatterino7/assets/169834686/786bdc97-a497-4d1a-9e52-5e0115c5e39a)

**Comment by Nerixyz** [comment]
This is a topic for upstream Chatterino. BTTV badges are discussed in https://github.com/Chatterino/chatterino2/discussions/4323. FFZ:AP badges were discussed in https://github.com/Chatterino/chatterino2/issues/2289.

**Reply by Atoxiv** [reply]
Alright, ty!
